### PR TITLE
Update the documentation in the standard format

### PR DESCRIPTION
**Pull request by Nuvindu**
# Description
$subject

### CODE REVIEW

**Review by RDPerera — DISMISSED** [review]
LGTM

**Review by RDPerera — APPROVED** [review]

File: examples/project-management-with-calendar/project-management-with-calendar.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Personal Project Management with Google Calendar API
+# Project management with Google Calendar API
 
 Let's explore how Alex, a software developer, leverages the Google Calendar API in Ballerina for managing his personal project schedule and collaborating with team members.
 
 ## Step 1: Import Google Calendar Connector
 
-Alex begins by importing the `ballerinax/googleapis.gcalendar` module into his Ballerina project.
+Alex begins by importing the `ballerinax/googleapis.gcalendar` package.
```
Comment (ThisaruGuruge) [inline examples/project-management-with-calendar/project-management-with-calendar.md:7]: Module

File: examples/work-schedule-management-with-calendar/work-schedule-management-with-calendar.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# Work Schedule Management with Google Calendar API
+# Work schedule management with Google Calendar API
 
 Sarah relies on the Google Calendar API to efficiently manage her work schedule. Her application interacts with the API for various tasks related to scheduling and organizing work-related events and meetings.
 
 ## Step 1: Import connector
 
-Import the `ballerinax/googleapis.gcalendar` module into the Ballerina project.
+Import the `ballerinax/googleapis.gcalendar` package.
```
Comment (ThisaruGuruge) [inline examples/work-schedule-management-with-calendar/work-schedule-management-with-calendar.md:7]: module

**Comment by RDPerera** [comment]
+1 with `module` instead of package

File: examples/project-management-with-calendar/project-management-with-calendar.md (hunk 1 above)
Comment (Nuvindu) [reply examples/project-management-with-calendar/project-management-with-calendar.md:7]: Fixed in a new pr https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/149/

File: examples/work-schedule-management-with-calendar/work-schedule-management-with-calendar.md (hunk 2 above)
Comment (Nuvindu) [reply examples/work-schedule-management-with-calendar/work-schedule-management-with-calendar.md:7]: Fixed in a new pr https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/149

**Pull request merged**